segmentation:
  ros__parameters:
    distance_threshold: 0.02
    max_avg_distance_from_sensor: 1.5
    max_points_segment: 100
    max_segment_width: 0.2
    method_threshold: diet # lee santos ''
    min_avg_distance_from_sensor: 0.05
    min_points_segment: 3
    min_segment_width: 0.04
    noise_reduction: 0.3
    restore_defaults: false
    scan_topic: scan_filtered
    segmentation_type: jump_distance_merge
    segments_topic: segments
